chainer/link_hooks/weight_standardization.py

```diff
@@ -0,0 +1,81 @@
+from chainer.functions.normalization._standardize import _standardize
+from chainer import link_hook
+
+
+class WeightStandardization(link_hook.LinkHook):
+    """Weight Standardization link hook implementation.
+
+    This hook standardizes a weight by *weight statistics*.
+
+    This link hook implements a "weight standardization" which computes the
+    mean and variance along axis "output channels", then normalize by these
+    statistics.
+    That improves training by reducing the Lipschitz constants of the loss and
+    the gradients like batch normalization but without relying on large batch
+    sizes during training. Specifically, in micro-batch training, weight
+    standardization significantly outperforms other normalization methods like
```

Comment:
Isn't Group Normalization necessary for Weight Standardization to overperform over BN?

Reply:
Oh, thank you, it has gone out while changing sentences sometimes.

Reply:
Thanks for update.